standards.md

```diff
@@ -1,47 +1,133 @@
 
-This means that most PRs should include both:
+_See also [the Tekton review process](https://github.com/tektoncd/community/blob/master/process.md#reviews)._
 
-1. Tests
-2. Documentation updates
+## Pull request description
 
-## Commit Messages
+* Include a link to the issue being addressed, but describe the context for the reviewer
+  * If there is no issue, consider whether there should be one:
+    * New functionality must be designed and approved, may require a TEP
+    * Bugs should be reported in detail
+  * If the template contains a checklist, it should be checked off
+  * Release notes filled in for user visible changes (bugs + features),
+    or removed if not applicable (refactoring, updating tests) (may be enforced
+    via the [release-note Prow plugin](https://github.com/tektoncd/plumbing/blob/master/prow/plugins.yaml))
 
-All commit messages should follow
-[these best practices](https://chris.beams.io/posts/git-commit/), specifically:
+## Commits
 
-- Start with a subject line
-- Contain a body that explains _why_ you're making the change you're making
-- Reference an issue number one exists, closing it if applicable (with text such
-  as
-  ["Fixes #245" or "Closes #111"](https://help.github.com/articles/closing-issues-using-keywords/))
+* Follow [commit messages best practices](https://chris.beams.io/posts/git-commit/):
+  1. Separate subject from body with a blank line
+  2. Limit the subject line to 50 characters
+  3. Capitalize the subject line
+  4. Do not end the subject line with a period
+  5. Use the imperative mood in the subject line
+  6. Wrap the body at 72 characters
+  7. Use the body to explain what and why vs. how. Don't just link to an issue and
+      [aim for 2 paragraphs](https://www.youtube.com/watch?v=PJjmw9TRB7s), e.g.:
+      * What is the problem being solved?
+      * Why is this the best approach?
+      * What other approaches did you consider?
+      * What side effects will this approach have?
+      * What future work remains to be done?
+* Prefer one commit per PR; if there are multiple commits, ensure each is
+  self-contained and makes sense without the context of the others in the PR
 
-Aim for [2 paragraphs in the body](https://www.youtube.com/watch?v=PJjmw9TRB7s).
-Not sure what to put? Include:
+## Docs
 
-- What is the problem being solved?
-- Why is this the best approach?
-- What other approaches did you consider?
-- What side effects will this approach have?
-- What future work remains to be done?
+* Include Markdown doc updates for user visible features
+* Spelling and grammar should be correct
+* Try to make formatting look as good as possible (use preview mode to check)
+* Follow [content](https://github.com/tektoncd/website/blob/master/content/en/doc-con-content.md)
+  and [formatting](https://github.com/tektoncd/website/blob/master/content/en/doc-con-formatting.md) guidelines
+* Should explain thoroughly how the new feature works
+* Ensure that all links and references are valid
```

Comment:
i think the docs should include not only a code snippet but also a reference to an end-to-end example that uses the feature

Reply:
yes. yes it should. :sweat_smile: 
https://github.com/tektoncd/pipeline/issues/1288

Reply:
Okay I've tried to capture this, let me know what you think! We can always add to it (esp. if we are able to add some code snippet testing automation one day... :D)